trimesh/io/wavefront.py

```diff
@@ -193,23 +202,29 @@ def append_mesh():
                 loaded['metadata']['face_groups'] = face_groups
 
             if len(current['usemtl']) > 0:
-                current_mtllibs = [mtllibs[k] for k in current['usemtl']
-                                   if k in mtllibs]
-                try:
-                    vertex_colors = _get_vertex_colors(
-                        uv=loaded['metadata']['vertex_texture'],
-                        mtllibs=current_mtllibs,
-                        resolver=resolver)
-                    loaded['vertex_colors'] = vertex_colors
-                    loaded['metadata']['mtllibs'] = current_mtllibs
-                except BaseException:
-                    log.error('failed to load texture!', exc_info=True)
+                texture = np.array(current['vt'], dtype=np.float64)
+                vertex_colors = np.zeros((0, 4), dtype=np.uint8)
+                for usemtl, findices in usemtl_to_findices.items():
+                    uv = texture[findices]
```

Comment:
This is the line killing the builds with:
`IndexError: index 5280 is out of bounds for axis 0 with size 882`

Reply:
8400f85 fixes this.